web/src/beta/components/fields/Property/PropertyField/index.tsx

```diff
@@ -154,6 +155,18 @@ const PropertyField: React.FC<Props> = ({
           onSave={handleChange}
           onFlyTo={onFlyTo}
         />
+      ) : schema.type === "array" ? (
+        schema.ui === "range" && (
```

Comment:
For me i prefer `schema.type ===  "array" && schema.ui === "range" ? <RangeField/> : <p/>` here. In this way it's easier for developer to notice if he add some schema in type of "array" but no ui / has other ui.

Reply:
You said done, but it doesn't seem like you have made these changes yet?

Reply:
Emmm i'm not sure whether i'm reading the latest code or not... but basically what i mentioned is I guess it's better to display a `<p key={schema.id}>{schema.name} field</p>` instead of nothing when a data with `type` "array" but no `ui` (or `ui` is not range) comes. Currently it will still become nothing displayed for this case.

Reply:
@airslice this is how it looks now is not right??
![image](https://github.com/reearth/reearth/assets/89770889/68410255-5352-4263-b612-dcc3c0795437)